### ISSUE TITLE
Get rid of this "Camera Smoothing)

### ISSUE THREAD

**Issue by DogeisCut** [Taureon/aps-plus-plus#7]
![image](https://github.com/Taureon/aps-plus-plus/assets/23178844/01d3573a-1c45-46c3-a2e5-f9e725d83e64)
This is apparently the result of a bug. From what I've heard, removing this breaks predator zoom.

**Comment by LostSoul6Years** [comment]
Fixed it. They didnt care.

**Comment by DogeisCut** [comment]
gonna talk to Taureon about this...

**Comment by LostSoul6Years** [comment]
just check my latest commit. My PR is taken very lightly.
Dont bother about it.

**Comment by LostSoul6Years** [comment]
The other dev said I removed the smoothing and dismissed the issue as "flipping the boolean" which made no actual sense to me. And said it was not of priority and he will take it into his own hands and "fix it" again. 

Then I kept the smoothing and now people are still like "what?". And now comes the mocking response with that. I am genuinely confused and concerned about how hard is it to understand my PR that completely addresses the healthbar rendering issue.

I do not know what I am being disrespected and taken lightly for, but that is clearly uncalled for.
